### ISSUE TITLE
Let's beat the state of the art once again! 

### ISSUE THREAD

**Issue by LifeIsStrange** [hiroki13/span-based-srl#6]
You are still the number one state of the art at semantic role labeling.
But in the meantime there was a lot of progress in the Nlp world. 
ELMO has been succeeded by BERT, bert has been succeeded by spanBert and all of them have been succeeded by XLnet!

There is a popular library for easily switching from one transformer to another:
https://github.com/huggingface/transformers

If you trained again your model, with XLnet instead of ELMO you would probably significantly increase accuracy at SRL!

Humanity *needs* more accurate SRL, please :}

**Comment by LifeIsStrange** [comment]
@hiroki13

**Comment by logicReasoner** [comment]
@LifeIsStrange  Yeah, we need even better SRL. Still too many dumb errors in the output of the current SOTA :(
Which model do you currently use?
